src/Symfony/Component/Translation/Translatable.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+/**
+ * @author Nate Wiebe <nate@northern.co>
+ */
+final class Translatable
```

Comment:
im wondering if it's worth adding __toString(), thus making it a stringable object.

just for pragmatic reasons :sweat_smile:

Reply:
If we were to add that, what would we return? I needs to be translated by a translator and isn't aware of the service. Potentially we could throw an exception with some help text to point the user in the direction of using a translator.

Reply:
i thought about the message ID,  but i agree it doesnt really mean anything as a string value on its own. Forget about it :)